### ISSUE TITLE
Warning since Microsoft.Azure.SignalR 1.1.0

### ISSUE THREAD

**Issue by cveld** [Azure/azure-signalr#673]
I tried to upgrade the Microsoft.Azure.SignalR package reference from version 1.1.0-preview1-10442 to 1.1.0. Unfortunately this results in the following warning:

```
Warning	NU1603	Microsoft.Azure.SignalR 1.1.0 depends on Microsoft.AspNetCore.Http.Connections.Client (>= 3.0.0-rc2.19460.8) but Microsoft.AspNetCore.Http.Connections.Client 3.0.0-rc2.19460.8 was not found. An approximate best match of Microsoft.AspNetCore.Http.Connections.Client 3.0.0 was resolved.
```

What is your guidance?

**Comment by JialinXin** [comment]
Thanks for reporting this. This is a version issue we just fixed in #670. And will be released in v1.1.1. No functional break between the 2 package of Microsoft.AspNetCore.Http.Connections.Client.

**Comment by JialinXin** [comment]
The issue is fixed in the [v1.1.1](https://github.com/Azure/azure-signalr/releases/tag/v1.1.1). Please update to resolve install package warnings.
